css/main.css

```diff
@@ -0,0 +1,117 @@
+*{
+    background-color: rgb(0, 0, 0);
+    font-family: "Roboto",sans-serif;
+    color: rgb(223, 48, 48);
+}
+.TB{
+    height: 12vw;
+}
+#camera{
+    display: inline;
+    padding-left:150px;
+}
+.orientation{
+    border:solid black;

```

Comment:
warning: Using width with border can sometimes make elements larger than you expect.

(I believe there's a `box-sizing: border-box` property or something along those lines which will make the border part of the size. Alternatively, if you're ok with this, you can just ignore this warning.)